fhir-path/src/main/java/com/ibm/fhir/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -1180,6 +1194,9 @@ public Collection<FHIRPathNode> visitFunction(FHIRPathParser.FunctionContext ctx
             case "ofType":
                 result = ofType(arguments);
                 break;
+            case "repeat":
```

Comment:
Which of these options is most likely? If repeat isn't likely to be used, then perhaps we can stick it later in the switch. Should perform a little better, I think (assuming Java implements this as a linear series of comparisons....I remember discussing with John ages ago, but can't remember what he found).

Reply:
I'm pretty sure its not linear comparisons.

Reply:
from https://docs.oracle.com/javase/8/docs/technotes/guides/language/strings-switch.html
> The Java compiler generates generally more efficient bytecode from switch statements that use String objects than from chained if-then-else statements.